rollup.js

```diff
@@ -46,18 +47,18 @@ module.exports = ({ name, jsBundle = 'bundle.js', cssBundle = 'bundle.css', inst
         const relativeOutputDirStart = outputDir.indexOf(OUTPUT_DIR_ROOT);
 
         if (relativeOutputDirStart === -1) {
-            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}'` });
+            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}', was: ${outputDir}` });
         }
 
         const relativeOutputDir = outputDir.substring(relativeOutputDirStart);
-        const finalOutputDir = `${relativeOutputDir}/${instrumentDir}`;
+        const finalOutputDir = path.join(relativeOutputDir, instrumentDir);
 
-        const processedHtml = html(name, finalOutputDir, imports, cssCode, jsCode);
+        const processedHtml = html(name, finalOutputDir.replace(/\\/g, "/"), imports, cssCode, jsCode);
```

Comment:
Every line in this file relates to filesystem paths. This is the only location where it turns into a HTML compatible file path, which is always with `/`. Thus, we need to convert it here.